test/optim/test_lrscheduler.py

```diff
@@ -1703,6 +1703,32 @@ def test_cycle_lr_with_adam(self):
         self._test_cycle_lr(scheduler, lr_targets, momentum_targets, 10, use_beta1=True)
         self.opt = old_opt  # set optimizer back to SGD
 
+    def test_onecycle_lr_pickleable(self):
+        adam_opt = Adam(self.net.parameters())
+        scheduler = OneCycleLR(adam_opt, max_lr=5, total_steps=200)
+        state = scheduler.state_dict()
+        self.assertNotIn("anneal_func", state)
+        pickle.dumps(state)
+
+    def test_onecycle_lr_restore_from_state_dict(self):
+        adam_opt = Adam(self.net.parameters())
+
+        # case 1: cosine annealing
+        scheduler = OneCycleLR(adam_opt, max_lr=5, total_steps=200, anneal_strategy='cos')
+        restored_scheduler = OneCycleLR(adam_opt, max_lr=5, total_steps=200)
+        restored_scheduler.load_state_dict(scheduler.state_dict())
+        self.assertTrue(restored_scheduler.anneal_strategy == scheduler.anneal_strategy == 'cos')
```

Comment:
Could we switch these to two assertEquals for better test error messaging if someone causes this test to fail in the future?